ATOM/Dependencies/Detectron/Optimizations/Disable startups.ps1

```diff
@@ -30,6 +30,7 @@ $startups = @(
 	"Battle.net",
 	"BrHelp",
 	"com.squirrel.Teams.Teams",
+	"CCleaner Smart Cleaning",

```

Comment:
Are you sure these registry keys are correct? They will be located in one of the four locations:
```
"HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run"
"HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run32"
"HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run"
"HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run32"
```

Reply:
Double checked to confirm, all are correct.